code-challenges.test.js

```diff
@@ -21,9 +23,60 @@
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// PSEUDO /////////////
+
+// The following paragraph is the blueprint to run the tests
+
+// describe("name of function",() => {
+//     it ("description of functionanlity what iy does", () => {
+//         expect ("invoke function").toEqual("expected result")
+//     })
+// }
+
+
+
+describe("fib", () => {
+    it("takes a number greater than 2 and returns an array", () => {
+        expect(fib(6)).toEqual([1, 1, 2, 3, 5, 8])
+    })
+})
+
+describe("fib", () => {
+    it("takes a number greater than 2 and returns an array", () => {
+        expect(fib(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+    })
+})
+
+
```

Comment:
These tests look great. The only edit is that you can put both your expect statements in the same test. 
```
describe("fib", () => {
    it("takes a number greater than 2 and returns an array", () => {
        expect(fib(6)).toEqual([1, 1, 2, 3, 5, 8])
        expect(fib(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
    })
})
```